schemas/Config.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "Class": {
+      "$ref": "#/definitions/Config"
+    }
+  },
+  "required": [
+    "Config"
+  ],
+  "title": "Config",
+  "definitions": {
+    "Class": {
+      "$id": "#/properties/Holiday",
+      "type": "object",
+      "title": "The Config schema",
+      "description": "All configurations stored in key value pair.",
+      "required": [],
+      "properties": {
+        "key": {
```

Comment:
For config let's use the below schema 
module (string)
key (string)
value (string)
context (string)

Remove schoolId

Reply:
@coolbung - updated the changes